bokehjs/src/coffee/api/plugin.coffee

```diff
@@ -0,0 +1,27 @@
+
+define [
+  "underscore"
+  "jquery"
+  "common/logging"
+  "./figure"
+  "./helpers"
+], (_, $, Logging, figure, helpers) ->
+
+  logger = Logging.logger
+  show = helpers.show
+
+  _api = {
+    "figure": figure
+  }
+
+  $.fn.bokeh = (type, args) ->
```

Comment:
> it is that case that all the examples do work using $().bokeh

So this should actually be true because jquery isn't a true/at all an AMD module, we just fake it with `noConflict` if jquery was already included in a page. The current approach doesn't make any harm, maybe besides confusing users. As this is alpha/experimental API, I would just leave it as-is and figure out it for the next release (so basically what you were saying in previous comment).